_posts/2021-01-08-2020-review.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: post
+title: Trino in 2020 - An amazing year in review
+author: Martin Traverso, Manfred Moser, Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+**Wow!** If you would have to sum up what happened in the last that year in this
+great community, that would be it. It is truly awe-inspiring to be part of this
+incredible journey of Trino. Oh, yeah on that note. Our community and project
+[chose the new name Trino]({% post_url 2020-12-27-announcing-trino %}), to be
+able to continue to innovate and develop freely as a community of peers. Presto®
+and Presto SQL are a thing of the past.
+
+Now that is out of the way, let’s dive right in, and see we all our community
+members across the globe have created with us!
+
+<!--more-->
+
+## By the numbers
+
+Even the size and growth of [our community on Slack](/slack.html) is impressive:
+
+* Started 2020 with ~1600 members and 280 weekly active
+* Over 3200 members
```

Comment:
.. at the end of the year, or "on 2021's Eve"

Reply:
Added stuff that captures that